app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -0,0 +1,79 @@
+
+We also have information for people outside of the UK who want to get [international qualified teacher status](/international-qualified-teacher-status) (iQTS) or [come to teach in England](/come-to-england-to-teach-if-you-are-a-teacher-from-outside-the-uk).
+
+##Study as a postgraduate student
+You can study for [a PGCE or PGDE](/what-is-a-pgce) (an academic qualification) at a university. This is one way to get QTS. Most of the time you will be in schools on placement but there is some academic learning. Courses typically take one year but you may be able to study part-time.
+
+Fees for full-time courses are usually around £9,250 but you can:
+
+- get [funding that you do not have to pay back](/funding-your-training#bursaries-and-scholarships) for certain subjects
+- [apply for a student loan](/funding-your-training#tuition-fee-and-maintenance-loans)
+
+Fees are higher for international students.
+
+You should check the fees for a particular course you are interested in with the course provider.
+
+##Train in school (fee paying)
+There are different programmes that focus on training you while you teach in a school:
+
+- School-centred initial teacher training (SCITT)
+- School Direct (fee-funded)
+
+SCITT programmes are delivered by a network of schools, providing school-based training. You won’t be paid a salary while you are training and will pay fees.
+
+School Direct (fee-funded) programmes are delivered by groups of schools in collaboration with a SCITT or university. The content and structure of a School Direct course is broadly similar to a SCITT course and they don't pay you a salary. Some SCITT and School Direct (fee-funded) programmes give you a PGCE or PGDE in addition to QTS. Courses typically take one year but you may be able to study part-time.
+
+Fees are usually around £9,250 but you can:
+
+- get [funding that you do not have to pay back](/funding-your-training#bursaries-and-scholarships) for certain subjects
+- [apply for a student loan](/funding-your-training#tuition-fee-and-maintenance-loans)
+
+You should check the fees for a particular course you are interested in with the course provider.
+
+##Earn a salary while you train
+These courses are usually only suitable for people who already have relevant teaching experience or are already working as unqualified teachers. You may be able to study part-time.
+
+[Read about Nathan’s experience of salaried teacher training](/my-story-into-teaching/teacher-training-stories/salaried-teacher-training-classroom-learning)
+
+###School direct (salaried)
+If you have the right experience, School direct (salaried) give you QTS with many courses also offering a PGCE or PGDE. It takes a year to complete.
+
+[Find out more about School direct](https://www.ucas.com/teaching-option/school-direct-salaried) (salaried)
```

Comment:
We shouldn't be linking to UCAS for postgraduate content

Reply:
I've linked to Find. There's no information on the current page. If there's a better place to link to we can update it.